MetricsUploader/TestClients/StartAndSendEventClient.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <cstdint>
+
+#include "MetricsUploader/Result.h"
+#include "orbit_log_event.pb.h"
+
+#define DllExport __declspec(dllexport)
+
+using orbit_metrics_uploader::OrbitLogEvent;
+using orbit_metrics_uploader::OrbitLogEvent_LogEventType_UNKNOWN_EVENT_TYPE;
+
+using orbit_metrics_uploader::Result;
+
+extern "C" {
+DllExport enum Result StartUploaderClient() { return Result::kNoError; }
```

Comment:
Is `enum Result` correct here? Shouldn't this be `int`? What does the header that is generated from the go-code show? Is there an enum?

Reply:
I put the same enum in the preamble, so generated header have the `enum Result` declaration. The signature of the functions are:
```
extern enum Result StartUploaderClient();
extern enum Result SendOrbitLogEvent(uint8_t* serializedProto, int length);
```